runtime/compiler/optimizer/J9EstimateCodeSize.cpp

```diff
@@ -325,14 +325,13 @@ static Block * getBlock(TR::Compilation *comp, Block * * blocks,
    if (!blocks[i])
       {
 
-      TR::TreeTop *startTree = TR::TreeTop::create(comp, TR::Node::createOnStack(
+      TR::TreeTop *startTree = TR::TreeTop::create(comp, TR::Node::create(
             NULL, TR::BBStart, 0));
-      TR::TreeTop *endTree = TR::TreeTop::create(comp, TR::Node::createOnStack(
+      TR::TreeTop *endTree = TR::TreeTop::create(comp, TR::Node::create(
             NULL, TR::BBEnd, 0));
 
       startTree->join(endTree);
-      blocks[i] = new (comp->trStackMemory()) Block (startTree, endTree,
-            comp->trMemory());
+      blocks[i] = Block::createBlock(startTree, endTree, cfg);
```

Comment:
I think you need a TR:: to make sure you get the most derived implementation here.

Reply:
So, in this case. It is indeed using `TR::Block`. There is a preprocessor macro that expands `Block` to `TR::Block` on [this line](https://github.com/eclipse/openj9/blob/9b2f7de10a9e2d1b1aa181ee6550314ae2cf1ba3/runtime/compiler/optimizer/J9EstimateCodeSize.cpp#L321).

I can remove the macro if you want and make the call to `TR::Block` explicit.

Reply:
hmm macro is ugly but predates your changes - so can be left for now I guess.